client/src/components/ForgotPassword/style.scss

```diff
@@ -0,0 +1,297 @@
+        }
+      }
+
+      @media screen and (max-width: $mobile-width) {
+        display: none;
+      }
+    }
+
+    .forgot-password-wrapper {
+      width: 100%;
+      max-width: 400px;
+      margin: 200px 0 auto;
+
+      ::placeholder {
+        font-family: Open Sans, serif;
+        font-size: 12px;
+        font-style: normal;
+        font-weight: 600;
+        line-height: 16px;
+        color: $ghost;
+      }
+
+      h2 {
+        font-family: Open Sans, serif;
+        font-size: 24px;
+        font-style: normal;
+        font-weight: 600;
+        line-height: 40px;
+        color: $black;
+
+        @media screen and (max-width: $mobile-width) {
+          margin-top: 20px;
+          font-size: 24px;
+          line-height: 24px;
+          color: $white;
+          text-align: center;
+        }
+      }
+
+      p {
+        width: 60%;
+        font-family: Open Sans, serif;
+        font-size: 14px;
+        font-style: normal;
+        font-weight: 600;
+        line-height: 19px;
+        color: $shuttle-gray;
+
+        @media screen and (max-width: $mobile-width) {
+          width: 100%;
+          margin: 20px 0 0;
+          color: $white;
+        }
+      }
+
+      .sign-up {
+        display: none;
+      }
+
+      .sign-in,
+      .sign-up {
+        margin: 12px 0 0;
+        font-family: Open Sans, serif;
+        font-size: 12px;
+        font-style: normal;
+        font-weight: 600;
+        line-height: 16px;
+        color: $black;
+        text-align: center;
+
+
+        @media screen and (max-width: $mobile-width) {
+          display: block;
+          margin-top: 15px;
+          color: $white;
+          text-align: left;
+        }
+      }
+
+      .email-wrapper {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        margin: 10px 0 0;
+
+        label {
+          width: 50%;
+          font-family: Roboto, serif;
+          font-size: 10px;
+          font-style: normal;
+          font-weight: 500;
+          line-height: 11px;
+          letter-spacing: 1px;
+          color: $river-bed;
+          text-transform: uppercase;
+
+          @media screen and (max-width: $mobile-width) {
+            margin-top: 20px;
+            color: $white;
+          }
+        }
+
+        input {
+          width: 100%;
+          height: 40px;
+          padding-left: 15px;
+          margin-bottom: 10px;
+          border: 1px solid $mischka;
+          outline: 0;
+          box-sizing: border-box;
+        }
+
+        a {
+          width: 50%;
+          font-family: Open Sans, serif;
+          font-size: 10px;
+          font-style: normal;
+          font-weight: 600;
+          line-height: 11px;
+          color: $havelock;
+          text-align: right;
+
+          @media screen and (max-width: $mobile-width) {
+            margin-top: 20px;
+            color: $white;
+          }
+        }
+
+        @media screen and (max-width: $mobile-width) {
```

Comment:
order of style should be next
1) main styles
2) styles in media queries
3) children, siblings and so on


so, in this case, it is hard to find selector for this media

Reply:
reworked